tests/test_tutorial/test_additional_responses/test_tutorial001.py

```diff
@@ -1,6 +1,5 @@
-from fastapi.testclient import TestClient
```

Comment:
We don't want to use isort in this tutorial, because `fastapi` are really third party here.

Reply:
I'm going to git reflog to the commit before this so probably your review will be lost. I have new changes to fix all this

Reply:
I think all of them use fastapi as third party... If that's true, yes.

Reply:
That's correct. I've made adjustment in the new isort configuration file. This should be flexible.